originals/t.yaml

```diff
@@ -76,13 +76,26 @@
   meta:
     genre: [Action, Adventure]
 
-- name: The Settlers II
+- name: [The Settlers, The Settlers (video game)]
+  names: [Die Siedler II, Serf City]
```

Comment:
is this right? The Settlers is also called "Die Siedler II"?

Reply:
It's right
https://www.mobygames.com/game/die-siedler-ii-die-nchste-generation-wikinger

Reply:
That's a good catch, it's Die Siedler and not Die Siedler II